src/WordPress/DataStore.php

```diff
@@ -63,6 +67,10 @@ public function getHostAPIUserUniqueId()
      */
     public function getClientV4APIKey()
     {
+        if (defined('CLOUDFLARE_API_KEY')) {
```

Comment:
for good measure, we probably want to also ensure the value isn't empty on the constant (in all three updates below too)

```suggestion
        if (defined('CLOUDFLARE_API_KEY') && CLOUDFLARE_API_KEY != "") {
```